homeassistant/components/tplink/entity.py

```diff
@@ -226,6 +226,7 @@ def __init__(
         """Initialize the entity."""
         self.entity_description = description
         super().__init__(device, coordinator, parent=parent, feature=feature)
+        self._async_call_update_attrs()
```

Comment:
Should this be in `async_added_to_hass` instead?

Reply:
Probably yes, the already merged platforms call this manually in their ctors, but they shouldn't need their own `__init__`. Maybe we should have a single PR that cleans this up after all are merged back in the `tplink_rewrite` to avoid fixing it separately for every individual platform PR?

Reply:
Considering this is the first PR to add that, I mvoed it into `async_added_to_hass` and update other PRs accordingly.